src/features/Main/Footer/index.jsx

```diff
@@ -9,17 +11,34 @@ export const Footer = () => {
   return (
     <footer>
       <ul className="footer-links">
-        <li>
+        <li className="footer-item">
+          <a href={termsOfServiceLink()}>
+            Terms
+          </a>
+        </li>
+        <li className="footer-item">
           <a href={privacyPolicyLink()}>
-            Privacy Policy
+            Privacy
+          </a>
+        </li>
+        <li className="footer-item">
+          <span className="icon">
+            <FontAwesomeIcon icon={faEarthAmericas} />
+          </span>
+          <a href={termsOfServiceLink()}>
+            English - US
           </a>
         </li>
-        <li>
+        <li className="footer-item">
+          <span className="icon">
+            <FontAwesomeIcon icon={faCookieBite} />
+          </span>
           <a href={termsOfServiceLink()}>
-            Terms of Service
+            Cookie preferences
```

Comment:
Why adding these items as they have no functionality? @AuraAlba

Reply:
Fixed, I added the link coming from configuration